units/getty@.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Getty on %I
+DefaultDependencies=no
+
+[Service]
+Environment=TERM=xterm-256color
+EnvironmentFile=/etc/getty/username.env
+ExecStart=-/sbin/agetty -o '-- ${USERNAME}' --noclear --skip-login - ${TERM}
+Type=idle
```

Comment:
One change worth experimenting with is whether Bottlerocket itself should run host container units with `Type=idle`, to ensure that we don't suppress normal boot output as soon as the admin container starts.

Reply:
It's currently not suppressing normal boot output even after the admin container starts. Not sure if that's desirable though:
```
[erikson@admin]$ [  119.581249] IPv6: ADDRCONF(NETDEV_CHANGE): eni43a75d0d3cf: link becomes ready
[  119.586160] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[  120.203507] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
...
```

Reply:
I'm wondering whether we still see output from e.g. systemd services starting after agetty starts, or if it just ends at the login prompt except for the occasional kernel message.

It might be hard to observe since the admin container takes a while to pull and start, but you could add a sleep to `kubelet` to see if we get both Starting + Started messages.

Reply:
I put a sleep in kubelet's `ExecPreStart` and I was able to observe systemd service messages for kubelet after I logged into the admin container:
```
[erikson@admin]$ [FAILED] Failed to start Kubelet.                                                                                 
See 'systemctl status kubelet.service' for details.                                                                                
[  OK  ] Reached target Multi-User System.                                                                                                                                                                                                                            
[erikson@admin]$ sudo sheltie 
```
Not quite sure why it failed after the sleep was done. It automatically restarted and fixed itself though.